### PR TITLE
Feat/225 code editor remove deprecated

### PR DESCRIPTION
**Pull request by gitdallas**
#225 

also updated replace prop generator so it properly shows usage if called without args

### CODE REVIEW

File: packages/pf-codemods/README.md [hunk 1]
```diff
@@ -172,6 +172,10 @@ Out:
 <ClipboardCopyButton position="bottom" />
 ```
 
+### codeeditor-remove-deprecated-props [(#8624)](https://github.com/patternfly/patternfly-react/pull/8624)
+
+We've removed the deprecated `entryDelay`, `exitDelay`, `maxWidth`, `position`, and `toolTipText` props. This rule will remove them from your code and suggest that you use the tooltipProps prop instead.
```
Comment (thatblindgeye) [inline packages/pf-codemods/README.md:177]: Just missing an examples section for in/output.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

**Review by thatblindgeye — APPROVED** [review]

File: packages/eslint-plugin-pf-codemods/test/rules/v5/codeEditor-remove-props.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/codeEditor-remove-props');
+
+ruleTester.run("codeEditor-remove-props", rule, {
+  valid: [
+    {
+      code: `import { CodeEditor } from '@patternfly/react-core'; <CodeEditor no />`,
```
Comment (wise-king-sullyman) [inline packages/eslint-plugin-pf-codemods/test/rules/v5/codeEditor-remove-props.js:7]: Not a blocker as I don't think there's any issues with it, but I'm curious what the reason is for the `no` here?

**Review by wise-king-sullyman — APPROVED** [review]
Looks good, one non-blocking question

File: packages/eslint-plugin-pf-codemods/test/rules/v5/codeEditor-remove-props.js (hunk 2 above)
Comment (gitdallas) [reply packages/eslint-plugin-pf-codemods/test/rules/v5/codeEditor-remove-props.js:7]: haha honestly idk, i may have used your generator and used "no" as the replacement value and forgot to remove here

**Pull request merged**